share/availability/window.go

```diff
@@ -11,8 +14,31 @@ const (
 // given AvailabilityWindow. If the window is disabled (0), it returns true for
 // every timestamp.
 func IsWithinWindow(t time.Time, window time.Duration) bool {
+	if alwaysAvailable {
+		return true
+	}
+	if windowOverride != time.Duration(0) {
+		window = windowOverride
+	}
 	if window == time.Duration(0) {
 		return true
 	}
 	return time.Since(t) <= window
 }
+
+// alwaysAvailable is a flag that disables the availability window. This flag is intended for
+// testing purposes only.
+var alwaysAvailable = os.Getenv("CELESTIA_PRUNING_DISABLED") == "true"
+
+// windowOverride is a flag that overrides the availability window. This flag is intended for
+// testing purposes only.
+var windowOverride time.Duration
+
+func init() {
+	durationRaw := os.Getenv("CELESTIA_OVERRIDE_AVAILABILITY_WINDOW")
+	duration, err := time.ParseDuration(durationRaw)
+	if err != nil {
+		panic("failed to parse CELESTIA_OVERRIDE_AVAILABILITY_WINDOW: " + err.Error())
+	}
+	windowOverride = duration
+}
```

Comment:
Seems like if CELESTIA_OVERRIDE_AVAILABILITY_WINDOW is set to `0` it can act as alwaysAvailable. Perhaps you can remove CELESTIA_PRUNING_DISABLED in favour of single env var?

Reply:
os.LookupEnv()